src/topics/command-line-tool.md

```diff
@@ -63,7 +66,70 @@ and in the [Outputs](../topics/outputs.md) sections.
 %
 % - Spaces in commands https://github.com/common-workflow-language/user_guide/issues/39
 % - Arguments (tell the reader the different use cases for arguments and inputs, tell them there is a section about inputs)
+Different use cases for arguments and inputs
```

Comment:
This seems to just be a phrase (not a full sentence)  instead of explanation of the differences

Reply:
Hello @swzCuroverse thank you for the feedback, i have made a commit with the differences and use cases